lib/galaxy/workflow/run_request.py

```diff
@@ -520,6 +563,18 @@ def add_parameter(name: str, value: str, type: WorkflowRequestInputParameter.typ
         "copy_inputs_to_history", "true" if run_config.copy_inputs_to_history else "false", param_types.META_PARAMETERS
     )
     add_parameter("use_cached_job", "true" if run_config.use_cached_job else "false", param_types.META_PARAMETERS)
+    for param in [
+        "preferred_object_store_id",
+        "preferred_outputs_object_store_id",
+        "preferred_intermediate_object_store_id",
+    ]:
+        value = getattr(run_config, param)
```

Comment:
Did you really mean to omit the default argument to getattr ?

Reply:
I meant run_config is a Python object (of type ``WorkflowRunConfig``) so all those properties should be defined. A null default wouldn't break anything but perhaps better to fail fast if the wrong type of object is passed in here?